src/libraries/Common/src/Interop/Windows/Advapi32/Interop.AllocateLocallyUniqueId.cs

```diff
@@ -9,6 +9,6 @@ internal static partial class Interop
     internal static partial class Advapi32
     {
         [DllImport(Libraries.Advapi32)]
```

Comment:
I don't know if that should be the goal for right now. The Analyzer/Fixer is designed to indicate to the user when it is appropriate to use the generator or when not to. I'm not a fan of making this the one true way because it isn't nor can it be. We specifically make decisions around the `GeneratedDllImport` not supporting some scenarios because they aren't appropriate for the most optimized path - consider the lack of `StringBuilder` optimization. We are also considering whether supporting `Delegate` is appropriate or if we should defer to `DllImport`.

Reply:
Move offline - 22.